pkg/ingester/rate.go

```diff
@@ -51,3 +51,8 @@ func (r *ewmaRate) tick() {
 func (r *ewmaRate) inc() {
 	atomic.AddInt64(&r.newEvents, 1)
 }
+
+// add counts some event.
```

Comment:
Not sure it's worth this comment. The function looks pretty straightforward.

Reply:
can i remove this line?

Reply:
I think so.